app/src/main/res/values-it/strings.xml

```diff
@@ -122,100 +122,162 @@
-    <string name="settings_about_join_c_jerboa">Iscriviti a c/jerboa</string>
-    <string name="settings_about_follow_on_mastodon">Segui su Mastodon</string>
-    <string name="settings_about_open_source">Open source</string>
+    <string name="settings_about_join_c_jerboa">Partecipa a c/jerboa</string>
+    <string name="settings_about_follow_on_mastodon">Seguici su Mastodon</string>
+    <string name="settings_about_open_source">Open Source</string>
     <string name="settings_about_source_code">Codice sorgente</string>
-    <string name="settings_about_source_code_subtitle_part1">"Jerboa è software libero e open-source, rilasciato sotto licenza "</string>
-    <string name="settings_about_source_code_subtitle_part2">GNU Affero General Public License v3.0</string>
-    <string name="account_settings_remove_current_avatar">Rimuovi avatar corrente</string>
-    <string name="account_settings_display_name">Mostra nome</string>
+    <string name="settings_about_source_code_subtitle_part1">"Jerboa è un software libero e open-source, rilasciato sotto licenza "</string>
+    <string name="settings_about_source_code_subtitle_part2"> GNU Affero General Public License v3.0</string>
+    <string name="account_settings_remove_current_avatar">Rimuovi L\'Avatar Attuale</string>
+    <string name="account_settings_display_name">Nome Visualizzato</string>
     <string name="account_settings_bio">Bio</string>
     <string name="account_settings_email">Email</string>
     <string name="account_settings_matrix_user">Utente Matrix</string>
     <string name="account_settings_avatar">Avatar</string>
     <string name="account_settings_banner">Banner</string>
-    <string name="account_settings_all">Tutte</string>
+    <string name="account_settings_all">Tutti</string>
     <string name="account_settings_local">Locali</string>
     <string name="account_settings_subscribed">Iscrizioni</string>
-    <string name="account_settings_default_listing_type">Tipologia predefinita</string>
+    <string name="account_settings_default_listing_type">Ordinamento Predefinito</string>
     <string name="account_settings_active">Attivi</string>
-    <string name="account_settings_hot">Di tendenza</string>
+    <string name="account_settings_hot">Popolari</string>
     <string name="account_settings_new">Nuovi</string>
-    <string name="account_settings_topday">Top giorno</string>
-    <string name="account_settings_topweek">Top settimana</string>
-    <string name="account_settings_topmonth">Top mese</string>
-    <string name="account_settings_topyear">Top anno</string>
-    <string name="account_settings_topall">Top di tutti i tempi</string>
-    <string name="account_settings_mostcomments">Più commenti</string>
-    <string name="account_settings_newcomments">Nuovi commenti</string>
-    <string name="account_settings_default_sort_type">Ordinamento predefinito</string>
+    <string name="account_settings_topday">TopGiorno</string>
```

Comment:
These are values for a drop-down. While I know that in English there's no spacing to separe multiple words at this location, I wonder if that's a glitch, and what would be the problem with using spaces to separate words here.

Reply:
Great! I wonder if you'd feel like opening a MR on the original (English) strings as well, maybe it's indeed a glitch by the dev ;)